### ISSUE TITLE
Make highway=track and bicycle=designated/yes more visible

### ISSUE THREAD

**Issue by LucasNicola** [cyclosm/cyclosm-cartocss-style#665]
In rural areas, there are lots of agricultural tracks (highway=track and tracktype=grade1) that are used a a cycle network (bicycle=designated or yes) as they are safe, comfortable, and already existing.
Unfortunately in cyclosm they are not rendered in blue like others cycleways and cyclepaths unless they are part of a cycleroute, which is not always the case. That makes them invisible to cyclists and it is a shame.
Could it be an improvement for the rendering ?
Thanks for reading me.

**Comment by Phyks** [comment]
Indeed, I think we only do it for highway=path or highway=cycleway (shared cycleways in the cities) but not highway=track.

Would you have some area which can be used for testing and is representative of the lack of render?

Thanks!

**Comment by LucasNicola** [comment]
Thanks for your reply! I live in Alsace, France and I'm thinking of this area South of Neuf-Brisach : 47.9885,7.5991
For instance there is a track with bicycle=designated going South to North from Nambsheim to Vogelgrun which is not rendered at all on cyclosm.
In Germany there are tons of them but most are part of a network so the render is good.
Let me know if I can help!

![Capture d’écran 2024-01-24 à 11 24 43](https://github.com/cyclosm/cyclosm-cartocss-style/assets/93993580/7fc1f595-d0ce-4812-bf26-84a1c8cd7a3d)
![Capture d’écran 2024-01-24 à 11 27 11](https://github.com/cyclosm/cyclosm-cartocss-style/assets/93993580/2c79c79e-ae96-420d-a2ae-e4fe9dc4af04)

**Comment by kaneap** [comment]
To add to this `highway=track` + `bicycle=designated` is already documented (on the[ German wiki](https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsanlagen_kartieren)):
![image](https://github.com/cyclosm/cyclosm-cartocss-style/assets/35740782/0367ce7a-09d4-4c12-a58b-f9d52d2ff941)

Should this situation be rendered the same  as `highway=path` + `bicycle=designated`? Perhaps the rendering for `highway=track` needs to be redone altogether, as we render access restrictions (e.g. `motor_vehicle=no` as light green) on other road types but not on highway=track.

**Comment by LucasNicola** [comment]
Hi, any update? Have you tested some render on an area?
Thanks!

**Comment by Phyks** [comment]
Sorry for my late reply, I've been quite busy lately and although I'm not forgetting this issue, I'm not sure when I will be able to commit enough time to tackle it. Happy to take any PR!

> To add to this highway=track + bicycle=designated is already documented (on the[ German wiki](https://wiki.openstreetmap.org/wiki/DE:Bicycle/Radverkehrsanlagen_kartieren)):

From my understanding (I can't speak German, relying on automated translation), this is not a regular `highway=track` but it requires further access restrictions tags (`access=agricultural|destination` for example). That being said, I think I would be fine to render a `highway=track` with limited motor vehicles traffic as a `highway=path` (and same thing with the subrender with explicit bicycle designation).

A `highway=track` with limited motor traffic should basically boil down to a `highway=path` which is large enough to fit a potential motor vehicle.

**Comment by Phyks** [comment]
Referencing #672 which is quite related.

**Comment by waterced** [comment]
> From my understanding (I can't speak German, relying on automated translation), this is not a regular highway=track but it requires further access restrictions tags (access=agricultural|destination for example). That being said, I think I would be fine to render a highway=track with limited motor vehicles traffic as a highway=path (and same thing with the subrender with explicit bicycle designation).

Should we check for restriction tags `access=agricultural|destination` or consider that they are implicit on a `highway=track`?
Similarly, should we color in blue only tracks with `bicycle=designated` or `bicycle=designated|yes`? To me, `bicycle=yes` is kinda implicit, I'll go for the first option, to only represent as cycleway the tracks that are explicitely designated for bicycles.

**Comment by Phyks** [comment]
Does anyone have ideas/opinions on what the render of a `highway=track` would be?

Here are my thoughts so far:

* `highway=track` as per [the wiki](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dtrack) : The tag [highway](https://wiki.openstreetmap.org/wiki/Key:highway)=track is used for minor land-access roads that are not considered part of the general-purpose road network [[1]](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dtrack#cite_note-1). Track roads are mostly used for agriculture, forestry, outdoor recreation, and similar activities on open land. Roads used for access to permanent human settlements or facilities should generally not use this tag.

* Therefore, `highway=track` is open to vehicles by default but should have very little traffic. Render should be comparable to `cycleway=lane` (therefore the dashed line at the moment) or bike roads in the city. However, dashed lines can also convey the meaning of worst quality/worst surface (which is also a common confusion about `highway=track` in OSM).

* I would consider it safe to assume `access=agricultural|destination` as being implicit on the tracks, from the wiki description of `highway=track` and from the [default access restrictions](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access_restrictions). Same for `bicycle=yes`.

* I don't get the meaning of explicit `bicycle=designated` on a `highway=track`?

* Maybe a middle ground would be to render `highway=track` with explicit `bicycle=designated` or `access=agricultural|destination` as if they were tagged as `highway=path`?

**Comment by waterced** [comment]
> ⇒ Maybe a middle ground would be to render highway=track with explicit bicycle=designated or access=agricultural|destination as if they were tagged as highway=path?

This is what I think would be the solution, I think `highway=track` & `bicycle=designated` would be a sufficient condition, let's display as if it was a `highway=path` & `bicycle=designated|yes`, with the same surface<->dash rules.


> I don't get the meaning of explicit bicycle=designated on a highway=track?

In my previous post, I put a streetview picture of a track that was chosen to be an official bike itinerary by local authorities, even though agricultural traffic is authorized, and materialized accordingly with signs. Such a way deserves `bicycle=designated` and a prefered rendering because it is expected to be more suitable for bikes than any random track in the countryside.

**Comment by Phyks** [comment]
> This is what I think would be the solution

I'll let this issue open a bit more to gather further feedbacks before implementation.


> In my previous post, I put a streetview picture of a track that was chosen to be an official bike itinerary by local authorities, even though agricultural traffic is authorized, and materialized accordingly with signs.

I'm not sure to agree, if the track is part of an official bike itinerary, the itinerary should be tagged. And it will convey naturally a special render as a bicycle route?

**Comment by waterced** [comment]
Indeed, yes if a track is part of an official itinerary, it can be part of a relation which it make it more visible.
But regardless of wether there is an official itinerary, some tracks are explicitely signed as cycleways, even though they are not part of any 'official itinerary'. The only thing preventing us to tag them as  `highway=cycleway` is wiki recommandations. Because `highway=cycleway` should be left for highways without motor vehicles. However, considering the low traffic involved, it seems reasonnable to me to display these tracks similarly to cycleways in CyclOSM. Hence my suggestion.

**Comment by quasart** [comment]
When there are many tracks in an area, and only one is optimised for bicycle, it would indeed be valuable to highlight it. 

Displaying them as cycleways sounds a good way to keep the map easy to read



Le 26 juin 2024 22:56:10 GMT+02:00, waterced ***@***.***> a écrit :
